src/code/bank2.asm

```diff
@@ -7323,7 +7323,7 @@ ENDC
     or   [hl]                                     ; $740C: $B6
     ld   hl, wIsBowWowFollowingLink               ; $740D: $21 $56 $DB
     or   [hl]                                     ; $7410: $B6
-    and  $7F                                      ; $7411: $E6 $7F
```

Comment:
Note that this most likely would have been better as `~BOW_WOW_KIDNAPPED`

Reply:
But there is also the possibility of bow wow being home. So I thought this would be more explicit.
Am I missing something?

Reply:
It is masking off the bit that says that bowwow is kidnapped

Reply:
I changed it, and I get warnings:
... -> src/code/entities/bank36.asm(624): [-Wtruncation]
Expression must be 8-bit

I can not make sense out of it and the docs do not help me out in this.

Reply:
Ah, yes... all math is done at 32bit. So that only works like `LOW(~BOW_WOW_KIDNAPPED)` or `BOW_WOW_KIDNAPPED ^ $FF`

Reply:
done with #443